app/controllers/db.js

```diff
@@ -23,6 +36,40 @@ module.exports = {
         fields: ['pin']
       }
     );
+  },
+  userAuthorization: function(id) {
+    return Brother.findById(id, {
+      include: [SysAdmin, Position]
```

Comment:
Can you point me to documentation around what this does? The sequelize docs are very hard to search through.